Makefile

```diff
@@ -2,7 +2,7 @@
 MKFILE_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 CPP_DIR := $(MKFILE_DIR)timemachine/cpp/
 INSTALL_PREFIX := $(MKFILE_DIR)timemachine/
-PYTEST_CI_ARGS := --cov=. --cov-report=term-missing
+PYTEST_CI_ARGS := --cov=. --cov-report=term-missing --durations=100
```

Comment:
Picked 100 as it gives a good number of timings, but doesn't grow with more tests. Let me know if there is a preferred value.

Reply:
Looks good to me-- also durations < 0.005s hidden by default